packages/lesswrong/components/sequences/SequencesPage.jsx

```diff
@@ -31,7 +31,8 @@ const styles = theme => ({
   title: {
     ...theme.typography.postStyle,
     fontVariant: "small-caps",
-    marginTop: 0
+    marginTop: 0,
+    fontFamily: theme.typography.uiSecondary.fontFamily,
```

Comment:
Same here, if we want to add this here, we should get rid of the postStyle, which does nothing but define a fontFamily.

Reply:
I think we should, I think it's mildly more clear what's going on, and it allows the EA Forum (and AF?) to change to sans serif in places like this in one place.

Reply:
Hmm, ok.